### PR TITLE
Added ARM64 Build

### PR DESCRIPTION
**Pull request by BlackMix**
**Description of the change:**
added arm64 on build

**Motivation for the change:**
#2686 and [kiali/issues#1091](https://github.com/kiali/kiali/issues/1091)
we need arm64 because the other repositories have added and are missing here.
from the tests I did it is already working.

### CODE REVIEW

**Comment by estroz** [comment]
This needs to be manually tested with an operator image.

File: .travis.yml [hunk 1]
```diff
@@ -183,6 +199,14 @@ jobs:
...
 
+    # Build and deploy arm64 scorecard-proxy docker image
+    - <<: *deploy
+      name: Docker image for scorecard-proxy (arm64)
+      arch: arm64
+      script:
+        - make image-build-scorecard-proxy
+        - make image-push-scorecard-proxy
+
     # Build and deploy amd64 scorecard-proxy docker image
```
Comment (camilamacedo86) [inline .travis.yml]: The Travis changes were verified here: https://travis-ci.org/github/camilamacedo86/operator-sdk/builds/668911026?utm_source=github_status&utm_medium=notification (See that these jobs are just executed against master and we need to do a fork pr to check them).

Screenshot 2020-03-31 at 00 11 10
Screenshot 2020-03-31 at 00 11 06
Screenshot 2020-03-31 at 00 11 02

c/c @joelanford

File: Makefile [hunk 2]
```diff
@@ -105,13 +105,15 @@ generate: gen-cli-doc gen-test-framework  ## Run all generate targets
 .PHONY: release_builds release
 
 release_builds := \
+    build/operator-sdk-$(VERSION)-aarch64-linux-gnu \
 	build/operator-sdk-$(VERSION)-x86_64-linux-gnu \
 	build/operator-sdk-$(VERSION)-x86_64-apple-darwin \
 	build/operator-sdk-$(VERSION)-ppc64le-linux-gnu \
 	build/operator-sdk-$(VERSION)-s390x-linux-gnu
```
Comment (camilamacedo86) [inline Makefile]: These tests need to be done manually by calling the release script and/or make release target. 
The image was created. See:

Screenshot 2020-03-31 at 00 13 03

c/c @joelanford

**Comment by joelanford** [comment]
It would be nice to check that the changes work in the Travis CI process as well.

We haven't documented the steps anywhere (but we should). I think they are:
1. Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
2. Set the following environment variables in the Travis settings for your fork:
    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
    - `HELM_IMAGE` same as above, but for helm
    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
    - `DOCKER_USERNAME` credentials for your repo
    - `DOCKER_PASSWORD` credentials for your repo
    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` (not sure if this is still required, depends on which version of Docker is running in Travis now)
3. Make a commit with `[travis deploy]` in the commit message on the branch with the aarch64 changes.
4. Check the travis build for your branch in your fork (not the PR build in the operator-sdk repo, since we don't allow PRs to build images in the `operator-framework` quay repo.)
5. Post a link in the PR to the Travis build page showing successful `Deploy` and `Deploy multi-arch manifest lists` stages with the new architecture.

@BlackMix if you're up for doing this, go for it. Otherwise, we have no problem doing this for you since we need to exercise these steps and add a doc for our own future reference.

**Comment by BlackMix** [comment]
@joelanford fine, thanks
I just did this, hoping to finish the deploy :)

**Comment by BlackMix** [comment]
well, I merged from master again and updated arm64 build for 
removed that fail because markdown tests

**Comment by BlackMix** [comment]
all builds success, but have some error in unit tests and not at build.
@joelanford @camilamacedo86 
[link from deploy](https://travis-ci.org/github/BlackMix/operator-sdk)

**Comment by camilamacedo86** [comment]
Hi @BlackMix, 

The error faced in https://travis-ci.org/github/BlackMix/operator-sdk/jobs/669087015 is because of the coveralls. You have not its TOKEN in your Travis env var configs. You can just comment the [line](https://github.com/operator-framework/operator-sdk/blob/master/.travis.yml#L134) and re-run again.

File: .travis.yml [hunk 3]
```diff
@@ -131,7 +131,7 @@ jobs:
         - (cd / && go get github.com/mattn/goveralls)
       script:
         - make test-sanity test-unit test-markdown
-        - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
+        # - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
```
Comment (camilamacedo86) [inline .travis.yml]: But we cannot do it in this pr commit. See that the issue faced in your Travis will not be faced here. 
You can add the ENV VAR TOKEN with an coveralls.io token to solve it in your Travis. 

OR 

Create a branch from it just to do the tests and then comment on this line.

**Comment by BlackMix** [comment]
@camilamacedo86 I added token later, but if I tell you that I was using the token from another repository and was giving an error because of that? my bad haha lol
I woke up now and I started putting it right
deploy now success [link](https://travis-ci.org/github/BlackMix/operator-sdk/builds/669385074)

**Comment by camilamacedo86** [comment]
Hi @joelanford,

See: https://travis-ci.org/github/BlackMix/operator-sdk/builds/669385074
WDYT?

**Comment by BlackMix** [comment]
will this be merged @joelanford ? so I and others community could use from the official

**Pull request merged**